Christina Kopecky

ckopecky opened the pull request.

ckopecky committed:
created a ReadMe markdown table with endpoints

created package.json file with start script

created server.js file with basic boilerplate

added gets for users, posts and tags

ckopecky committed:
typo in readme

ckopecky committed:
update readme

ckopecky committed:
one more update

ckopecky committed:
corrected a typo in 02-users.js

updated checklist table in readme

added error handling to server on what I have so far

ckopecky committed:
renamed 'done' to route handler

ckopecky committed:
added gets for user/:id, post/:id, tag/:id

ckopecky committed:
added post tags

ckopecky committed:
errors added for individual get user, get post, get id

ckopecky committed:
delete handlers added; updated checklist

ckopecky committed:
tested get - works OK

tested post - throws error 500 but it shows it worked when

perform a new get - weird

need another set of eyes to look to see where my error is.

designerexpert commented on server.js:
```diff
@@ -0,0 +1,210 @@
+    userDb
+        .insert({ name })
+        .then(res =>{
+           console.log(res.status(201).json(res));
```
remove the console.log in this line:
Consider replacing with just: ```res.status(201).json(ressult)```

ckopecky committed:
took out console.log on line 76

designerexpert commented on server.js:
```diff
@@ -0,0 +1,210 @@
+    }
+    userDb
+        .insert({ name })
+        .then(res =>{
```
rename res to something like: ```result```

ckopecky committed:
made minor corrections to code

lines 75-76

changed res to result

did the same for posts and tags

ckopecky committed:
posts by user done

post tags by ID is buggy - not sure what's wrong.

will take a break and come back to it.

ckopecky committed:
getting posts by User ID and tags by Post Id works!

ckopecky committed:
added puts to my checklist

designerexpert commented:
# Comments
* Great progress with those endpoints! Tons of reps on this project.
* Glad to see you got Posts to work well!
* Drive On with Puts and see how it works. Hint: ID comes from the routes usually while the object is included in the body as json.

ckopecky committed:
entered put requests

ckopecky committed:
checked off put requests

ckopecky committed:
create-react-app

added list of chars to app with some styling

ckopecky committed:
got rid of some unneeded files

ckopecky committed:
styling and added components

ckopecky committed:
react router implemented

Header page done

cast of chars listed on one page.

Buttons take you back and forth from one page to the next

ckopecky committed:
Update from upstream master

ckopecky committed:
remove client app to start over

ckopecky committed:
MVP complete backend

ckopecky committed:
create react app

ckopecky committed:
update middleware logger to be global

ckopecky committed:
update yarn

ckopecky committed:
add error handler for add

ckopecky committed:
update user to post and change body.name to body.text

ckopecky committed:
add .message to error so message is more helpful
